utils/zstd/Makefile

```diff
@@ -64,12 +64,6 @@ endif
 MESON_ARGS += \
 	-Dlegacy_level=7 \
 	-Ddebug_level=0 \
-	-Dbacktrace=false \
-	-Dstatic_runtime=false \
-	-Dbin_programs=true \
-	-Dbin_tests=false \
-	-Dbin_contrib=false \
-	-Dmulti_thread=enabled \
```

Comment:
I wonder, why we are removing this? Maybe it is matching the defaults, but anytime defaults can be changed and we do have at least some control about this.

cc: @neheb  - What do you think about this?

Reply:
I wrote it to have all the options explicit. I have no real opinion on whether or not that should change.

Reply:
IMO having smaller and simpler packages is what makes them easier to maintain and keeping up to date. Duplicating default values goes against KISS principle and in the end makes a package more brittle, #22308 is a perfect example of this.
Of cause everyone is free to choose their habits and I don't want to spend your time with my cheap rants. But if we want solve the problem of outdated zstd in OpenWrt, I could step in as its maintainer and solve possible future issues with it - empty PKG_MAINTAINER suggests that this role is vacant.
Cheers!